running locally/docker-prestissimo/prestissimo/workers/1/config_velox.properties

```diff
@@ -0,0 +1,4 @@
+coordinator=false
+http-server.http.port=8081
+discovery.uri=http://172.19.0.4:8080
+presto.version=0.282-fadfd95
```

Comment:
please rename this file to just config.properties

Use the version 0.282.
Any reason to use 8081? The yaml file does not expose it to the host.

Reply:
Changed presto.version to be 0.282 for both the worker's config.properties and coordinator's config.properties since both have to match for it to work.

Also changed 8081 to 8080.